distributions/otelcol-k8s/manifest.yaml

```
@@ -0,0 +1,71 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/k8s
+  name: otelcol-k8s
+  description: OpenTelemetry Collector for Kubernetes
+  version: 0.96.0
+  output_path: ./_build
+  otelcol_version: 0.96.0
```

```suggestion
  version: 0.97.0
  output_path: ./_build
  otelcol_version: 0.97.0
```
Is there a reason these are `0.96.0`? All the components are `0.97.0`.